modules/boost.config/1.86.0/overlay/BUILD.bazel

```diff
@@ -0,0 +1,16 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+cc_library(
+    name = "boost.config",
+    hdrs = glob(
+        [
+            "include/**/*.hpp",
+            "include/**/*.h",
+        ],
+    ),
+    features = [
+        "parse_headers",
```

Comment:
Some of the headers are not meant to be included and should be moved into `textual_hdrs`.